### PR TITLE
Milestone2

### PR DESCRIPTION
**Pull request by yahir91**
In this milestone, I created:

- Four javascript files inside the components folder.
- In each file, I created a stateless functional function.
- I build a static calculator template.

### CODE REVIEW

**Review by SpaYco — CHANGES_REQUESTED** [review]
# Changes Requested 

Hello! 👋 
Awesome job so far!! 👍 you still need to make some changes though! 

## Required Changes ♻️ 

- [ ] The `Button` component should display only 1 button, not multiple, the props shouldn't be an array of strings, but instead a string (prop should be called "name") ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/button/validateAsString.png))
- [ ] in your `ButtonPanel` component, instead of making arrays inside the button, wrap each row in a div and have each button use the `Button` component individually. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/buttonPanel/buttons.png))

### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

File: src/components/Button.js [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
   <div>
-    {buttons.map(symbol => (
-      <button key={symbol.toString()} type="button" className="buttons">
+    {name.map(symbol => (
+      <button key={symbol} type="button" className="buttons">
         {symbol}
       </button>
     ))}
   </div>
 );
```
Comment (SpaYco) [inline src/components/Button.js]: - [ ] you shouldn't use the `map` function here, this component should receive 1 string containing the name of the button, and not multiple strings in an array.

File: src/components/ButtonPanel.js [hunk 2]
```diff
@@ -1,16 +1,24 @@
...
+    <div className="main">
+      <>
+        <Button name={group1} />
+        <Button name={group2} />
+        <Button name={group3} />
+        <Button name={group4} />
+        <Button name={group5} />
+      </>
+    </div>
+  );
```
Comment (SpaYco) [inline src/components/ButtonPanel.js]: - [ ] why are you using the [`React.fragment`](https://reactjs.org/docs/fragments.html) if you're wrapping the component in a div? `React.fragment` is supposed to make your components return elements with nothing wrapping them, so wrapping it in a div destroys its purpose, either use the div or the fragment.

- [ ] Use divs here to create multiple rows of `Button` components, example: 

```jsx
<div>
    <Button name"AC" >
    <Button name"+/-" >
    <Button name"%" >
    <Button name"/" >
  </div>
  <div>
    <Button name"7" >
    <Button name"8" >
    <Button name"9" >
    <Button name"X" >
  </div>
  <div>
    <Button name"4" >
    <Button name"5" >
    <Button name"6" >
    <Button name"-" >
  </div>
  <div>
    <Button name"1" >
    <Button name"2" >
    <Button name"3" >
    <Button name"+" >
  </div>
  <div>
    <Button name"0" >
    <Button name"." >
    <Button name"=" >
  </div>
```

**Review by SpaYco — CHANGES_REQUESTED** [review]
# Changes Requested 

Hello Team! 👋 
Awesome job so far!! 👍 you still need to make some changes though! 

## Required Changes ♻️ 

sorry if it wasn't clear earlier, I'll try to describe the issue better below 👇 

### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Review by BrittanyBlake — APPROVED** [review]
### Status: Approved ✅ 
****
### Hi Team! 👋 
### Great job implementing the requested changes!! 🎖️ It's a 👍 from me! Go ahead and merge this pull request and keep up the good work 😄
****


 Good Luck and Happy Coding! 🍀 👩🏾‍💻

**Pull request merged**